test/cmdlineTests/model_checker_timeout_all/err

```diff
@@ -1,50 +1,36 @@
-Warning: CHC: Assertion violation might happen here.
- --> model_checker_timeout_all/input.sol:6:3:
+Warning: CHC: Division by zero might happen here.
+ --> model_checker_timeout_all/input.sol:6:11:
   |
-6 | 		assert(x > 0);
-  | 		^^^^^^^^^^^^^
+6 | 		require(x % 7 == 0);
```

Comment:
How can division by zero happen here?

Reply:
Is this because of the timeout?

Reply:
every mod and div expression check for that

Reply:
Yes, the query fails because of the timeout

Reply:
that's why it says "might"

Reply:
So can't the test cause problems here, i.e., the solver can figure out that division by zero cannot happen here?

Would it be better to increase the time so that the division by zero error message dissapears?